Sources/MapboxDirectionsCLI/ProcessCommand.swift

```diff
@@ -0,0 +1,95 @@
+
+import Foundation
+import MapboxDirections
+import SwiftCLI
+
+
+private let BogusCredentials = DirectionsCredentials(accessToken: "pk.feedCafeDadeDeadBeef-BadeBede.FadeCafeDadeDeed-BadeBede")
+
+class ProcessCommand<ResponceType : Codable, OptionsType : DirectionsOptions > : Command {
+    var name = "process"
+    
+    @Key("-i", "--input", description: "Filepath to the input JSON.")
+    var inputPath: String?
+    
+    @Key("-c", "--config", description: "Filepath to the JSON, containing serialized Options data.")
+    var configPath: String?
+    
+    @Key("-o", "--output", description: "[Optional] Output filepath to save the conversion result. If no filepath provided - will output to the shell.")
```

Comment:
The tool would be more generally useful if it outputs a pretty-printed summary of the route model by default. That pretty-printed output doesn’t need to be part of this PR, but for forwards-compatibility, perhaps we can introduce a `--format` flag that defaults to `json` for now but `text` later.

Reply:
Added a quick implementation, suitable for future enhancement.